changes/tests.py

```diff
@@ -260,56 +278,54 @@ def mock_get_all_messagesets():
     )
 
 
-def mock_get_active_subs_mc(registrant_id):
-    momconnect_prebirth_sub_id = "subscriptionid-momconnect-prebirth-0"
+def mock_get_subscriptions(querystring=None, results=[]):
+    url = 'http://sbm/api/v1/subscriptions/'
+    if querystring is not None:
+        url = url + querystring
```

Comment:
We can replace this with `url = "http://sbm/api/v1/subscriptions/{}".format(querystring)` which handles the case where querystring is `None`